test/util_spec.js

```diff
@@ -0,0 +1,341 @@
+import {expect} from 'chai';
+import deepFreeze from 'deep-freeze';
+
+import {
+  addTagToFilter,
+  capitalize,
+  cleanseTags,
+  filterCourses,
+  filterLessons,
+  fixNonArrayTagList,
+  lessonHasTags,
+  mergeObjects,
+  removeTagFromFilter
+} from '../src/util';
+
+describe('util', () => {
+
+  describe('capitalize', () => {
+
+    it('makes first letter in a string capital', () => {
+      const text = 'a random string';
+
+      expect(capitalize(text)).to.equal('A random string');
+    });
+  });
+
+  describe('getTags', () => {
+
+  });
+
+  describe('getCourses', () => {
```

Comment:
Jeg sliter litt med å finne ut hvordan getTags og getCourses kan testes, fordi de er avhengige av hva som finnes i ../oppgaver. Finnes det en måte å lage en slags 'mock' for ../oppgaver?